tools/WinObjC.Language/WinObjC.Language.nuproj

```diff
@@ -58,16 +58,14 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'" />
   <ItemGroup>
-    <None Include="project.json" />
     <None Include="WinObjC.Language.Packaging.targets" />
     <None Include="WinObjC.Language.props" />
     <None Include="WinObjC.Language.targets" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="Readme.txt">
-      <IncludeInPackage>true</IncludeInPackage>
+    <PackageFile Include="Readme.txt">
```

Comment:
>PackageFile [](start = 5, length = 11)

change this back to None. None items show up in the IDE whereas PackageFile do not. This was done so that the readme was easily visible and editable from within VS.

Reply:
(same for all packages you did this in)

---
In reply to: [141463510](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141463510) [](ancestors = 141463510)

Reply:
Using None here caused multiple readmes to get packaged, actually, due to Inference.

Reply:
(and throw error NG2001)